Tests/priority_queue_test.py

```diff
@@ -16,12 +16,13 @@ def test_insertion(self):
     def test_proper_order(self):
         """Test that a mixed-bag of priorities comes out in the right order."""
         queue = PriorityQueue()
-        items = [1, 2, 3]
-        priorities = [Priorities.LOW, Priorities.MEDIUM, Priorities.HIGH]
+        items = [1, 2, 3, 4]
+        priorities = [Priorities.LOW, Priorities.LOW, Priorities.MEDIUM,
+            Priorities.HIGH]
         for item, priority in zip(items, priorities):
             queue.push(priority, item)
 
-        items.reverse()
+        items = [4, 3, 1, 2]
```

Comment:
items = items[::-1]

Reply:
Its not exactly a reversal of the list. Notice that 1 and 2 have been swapped. The list indicates the expected ordering of the items.

Reply:
I see